react/Checkbox/Checkbox.test.js

```diff
@@ -1,126 +1,61 @@
-import chai, { expect } from 'chai';
-import sinon from 'sinon';
-import sinonChai from 'sinon-chai';
 import React from 'react';
-import {
-  Simulate,
-  renderIntoDocument,
-  scryRenderedDOMComponentsWithClass,
-  findRenderedDOMComponentWithClass
-} from 'react-dom/test-utils';
-import { createRenderer } from 'react-test-renderer/shallow';
-import SyntheticEvent from 'react-dom/lib/SyntheticEvent';
-import { findAllWithClass } from 'react-shallow-testutils';
-import Checkbox from './Checkbox';
-
-chai.use(sinonChai);
-
-const eventMatcher = sinon.match.instanceOf(SyntheticEvent);
+import { shallow } from 'enzyme';
 
-const renderer = createRenderer();
+import Checkbox from './Checkbox';
 
 describe('Checkbox', () => {
-  let element, checkbox, rootElement, input, label, errors, standard, button;
-
-  beforeEach(() => {
-    errors = [];
-
-    sinon.stub(console, 'error', errorMessage => {
-      errors.push(errorMessage);
-    });
-  });
-
-  afterEach(() => {
-    console.error.restore();
-  });
-
-  function render(jsx) {
-    element = jsx;
-    checkbox = renderer.render(element);
-    input = findAllWithClass(checkbox, 'input')[0] || null;
-    label = findAllWithClass(checkbox, 'label')[0] || null;
-    standard = findAllWithClass(checkbox, 'standard')[0] || null;
-    button = findAllWithClass(checkbox, 'button')[0] || null;
-  }
+  const simpleProps = {
```

Comment:
Right, I didn't understand that so perhaps it would be clearer if we just called it required props so you don't need to know that simple = required